respy/tests/test_f2py.py

```diff
@@ -544,50 +685,83 @@ def test_5(self):
             num_types,
         )
 
-        args = ()
-        args += base_args + (edu_spec, optim_paras)
-        py = pyth_contributions(*args)
+        py = pyth_contributions(
+            state_space, simulated_data, *shared_args, edu_spec, optim_paras
+        )
 
-        args = ()
-        args += base_args + (edu_spec["start"], edu_spec["max"])
-        args += (shocks_cholesky, delta, type_spec_shares, type_spec_shifts)
-        f2py = fort_debug.wrapper_contributions(*args)
+        f2py = fort_debug.wrapper_contributions(
+            periods_rewards_systematic,
+            mapping_state_idx,
+            periods_emax,
+            states_all,
+            data_array,
+            *shared_args,
+            edu_spec["start"],
+            edu_spec["max"],
+            shocks_cholesky,
+            delta,
+            type_spec_shares,
+            type_spec_shifts,
+        )
 
         assert_allclose(py, f2py)
 
         # Evaluation of criterion function
         x0 = get_optim_paras(optim_paras, num_paras, "all", is_debug)
 
-        base_args = (
+        base_args_1 = (
             is_interpolated,
             num_draws_emax,
             num_periods,
             num_points_interp,
             is_myopic,
             is_debug,
-            data_array,
+        )
+
+        base_args_2 = (
             num_draws_prob,
             tau,
             periods_draws_emax,
             periods_draws_prob,
+        )
+        base_args_3 = (num_agents_est, num_obs_agent, num_types)
+
+        py = pyth_criterion(
+            x0,
+            *base_args_1,
+            simulated_data,
+            *base_args_2,
+            state_space,
+            *base_args_3,
+            edu_spec,
+        )
+
+        f2py = fort_debug.wrapper_criterion(
+            x0,
+            *base_args_1,
+            data_array,
+            *base_args_2,
             states_all,
-            states_number_period,
+            state_space.states_per_period,
             mapping_state_idx,
             max_states_period,
-            num_agents_est,
-            num_obs_agent,
-            num_types,
+            *base_args_3,
+            edu_spec["start"],
+            edu_spec["max"],
+            edu_spec["share"],
+            type_spec_shares,
+            type_spec_shifts,
+            num_paras,
         )
 
-        args = base_args + (edu_spec,)
-        py = pyth_criterion(x0, *args)
-
-        args = base_args + (edu_spec["start"], edu_spec["max"], edu_spec["share"])
-        args += (type_spec_shares, type_spec_shifts, num_paras)
-        f2py = fort_debug.wrapper_criterion(x0, *args)
-
         assert_allclose(py, f2py)
 
+    @pytest.mark.skip(
```

Comment:
Is it ok to postpone it for now?

Reply:
Yes, please ensure an informative skip-message and move on.